docker/server/Dockerfile

```diff
@@ -1,5 +1,15 @@
 FROM node:18.12.1
 
+# ARGS
+ARG pgUser=postgres
+ARG pgPassword
+ARG pgHost=database
```

Comment:
future improvement: to make it easier, I usually like to set the args default from the .env file, then the args are just optional overrides

Reply:
Can you explain how you do that?